### PR TITLE
read data in from multiple csv files

### PR DESCRIPTION
**Pull request by BoltzmannBrain**

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@scottpurdy please review

File: fluent/utils/csv_helper.py [hunk 1]
```diff
@@ -57,6 +57,26 @@ def readCSV(csvFile, sampleIdx, numLabels):
     print e
 
 
+def readDir(dirPath, sampleIdx, numLabels):
+  """
+  Reads in data from a directory of CSV files; assumes the directory only
+  contains CSV files.
+  
+  @param dirPath            (str)          Path to the directory.
```
Comment (scottpurdy) [inline fluent/utils/csv_helper.py]: I still think lining things up like this violates the spirit of PEP 8 but seems fairly common now.

File: fluent/utils/csv_helper.py [hunk 2]
```diff
@@ -57,6 +57,26 @@ def readCSV(csvFile, sampleIdx, numLabels):
...
+  @param sampleIdx          (int)          Column number of the text samples.
+  @param numLabels          (int)          Number of columns of category labels.
+  
+  @return dataDict          (defaultdict)  Keys are CSV names, values are
+      OrderedDicts, where the keys/values are as specified in readCSV().
+  """
+  dataDict = defaultdict(list)
+  for _, _, files in os.walk(dirPath):
+    for f in files:
+      dataDict[f] = readCSV(os.path.join(dirPath, f), sampleIdx, numLabels)
```
Comment (scottpurdy) [inline fluent/utils/csv_helper.py]: what if two files in different dirs have the same name?

**Comment by BoltzmannBrain** [comment]
integrated into #116

**Comment by scottpurdy** [comment]
Why did you combine into another PR? It is way easy to review smaller PRs

**Comment by BoltzmannBrain** [comment]
True, but the main change in this PR (adding `readDir()`) was needed as #116 was being developed, so we implemented it there. Any revisions needed from your review here would be reflected in that implementation of `readDir()`.

**Pull request closed without merging**